delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/ResolversDummy.scala

```diff
@@ -155,6 +160,24 @@ class ResolversDummy private (
         _          <- cache.setToCache(res)
       } yield res
     }
+
+  private def verifyPriorityUniqueness(
```

Comment:
It belongs more to the state machine to do this check by providing a function to search resolvers.

Reply:
We also need to disable it for migration by checking if the env variable MIGRATE_DATA is defined

Reply:
In the state machine we don't have access to the cache. Besides that, for consistency with the storages, this is done [in the Impl section](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala#L80)

The migration part is not tacked in this PR

Reply:
For Realms, it is handled in the state machine for example. And for others, we don't pass the cache but other modules and so on.
For the migration part, it can be handled elsewhere, we just need not to forget about it.